tsconfig.json

```diff
@@ -9,7 +9,7 @@
         "allowSyntheticDefaultImports": true,
         "baseUrl": "src",
         "outDir": "dist",
-        "noUnusedLocals": true
+        "noUnusedLocals": false
```

Comment:
We already check for it in esLint and as far as I can tell there is no way to disable on a per-line basis (https://github.com/artsy/reaction-force/pull/162/files#diff-b7c8554bab28defe2bb4f7c32a1e2d33R68)